test/tests/bcs/dmg_periodic/tests

```diff
@@ -12,12 +12,9 @@
   [../]
 
   [./check_one_step]
-    type = 'Exodiff'
+    type = 'RunApp'
```

Comment:
Still have diff issues?

Reply:
I don't think so but maybe I didn't change it back to `Exodiff` because it seems like the purpose of this test (based on the requirement) is just to make sure that the boundaries are actually distributed? And there are a lot of other `Exodiff` tests in this directory

Reply:
Yes